jcl/src/java.management/share/classes/java/lang/management/RuntimeMXBean.java

```diff
@@ -130,6 +130,17 @@
 	 */
 	public String getName();
 
+	/*[IF Java10]*/
+	/**
+	 * Returns the process ID (PID) of the current running Java virtual machine.
+	 * 
+	 * @return the process ID of the current running JVM
+	 */
+	default long getPid() {
+		return ProcessHandle.current().pid();
```

Comment:
This API doesn't throw `SecurityException` like `ProcessHandle.current()`, so security check need to be stopped here.